apps/prairielearn/python/prairielearn.py

```diff
@@ -191,7 +190,9 @@ def get_enum_attrib(
     accepted_names = {member.name.replace("_", "-") for member in enum_type}
 
     if upper_enum_str not in accepted_names:
-        raise ValueError(f"{enum_val} is not a valid type")
+        raise ValueError(
+            f"{enum_val} is not a valid type, must be one of: {', '.join(str(member.value) for member in enum_type)}."
```

Comment:
```suggestion
            f"{enum_val} is not a valid type, must be one of: {', '.join(member.name.lower().replace("_", "-") for member in enum_type)}."
```
I think there's a somewhat simpler way of doing this by changing how accepted_names is computed.

Reply:
Done.

I think that way is more correct because it doesn't require the string value to actually match the member name. 

By the way, when you have strings within a format string you have to use the other type of quotes than the quotes used for the format strings (so `"_", "-"` needs to be `'_', '-'` in this case).